src/metiq.py

```diff
@@ -1202,6 +1274,12 @@ def main(argv):
             path = f"{outfile}.all.measurement.quality.csv"
         all_quality_stats.to_csv(path, index=False)
 
+    if len(all_frame_duration) > 0:
+        path = f"all.frame_duration.csv"
+        if outfile is not None and len(outfile) > 0:
+            path = f"{outfile}.all.frame_duration.csv"
```

Comment:
using a constant file name will be a problem: I think we prefer to exit with an error code here